rackcity/utils/asset_utils.py

```diff
@@ -0,0 +1,387 @@
+from django.core.exceptions import ObjectDoesNotExist
+from rackcity.models import (
+    Asset,
+    AssetCP,
+    NetworkPort,
+    NetworkPortCP,
+    PowerPort,
+    PowerPortCP,
+    PDUPort,
+    PDUPortCP,
+)
+from rackcity.models.asset import get_assets_for_cp
+from rackcity.utils.exceptions import (
+    MacAddressException,
+    NetworkConnectionException,
+    PowerConnectionException,
+)
+
+
+def save_network_connections(asset_data, asset_id, change_plan=None):
```

Comment:
Entire file: 3. Moved asset view helper methods to new utils file: `/utils/asset_utils.py`